### DISCUSSION TITLE
Unable to Upload a CSV file

### DISCUSSION THREAD

**Discussion by jgserran** [broadinstitute/seqr#2321] [Q&A]
In the "Edit Families & Individuals" tab under "Bulk edit individuals" I am unable to upload a .csv file to add new families into Seqr. Usually I see a blue box that lets me know how many families are being uploaded. However, I keep getting a message that the file is not uploading. I have attached images. 
<img width="1002" alt="Screen Shot 2021-11-22 at 2 27 23 PM" src="https://user-images.githubusercontent.com/89798273/142938386-134a0bd4-6a41-408d-8ebb-235d18267e44.png">
<img width="997" alt="Screen Shot 2021-11-22 at 2 27 38 PM" src="https://user-images.githubusercontent.com/89798273/142938394-e03f02d1-aeb4-4239-a498-cda81149c6b3.png">

**Comment by hanars** [comment]
Can you give me a little more data about the system you are running? Are you sure you are running the latest version of the code base?

**Reply by jgserran** [reply]
I am trying to upload an export of weekly surveys submitted through RGP. However, I think there might be an issue with the data within the spreadsheet. I am able to upload other spreadsheets from previous weeks, so I checked the column headers between the two and they match. Is it possible that there is something within the spreadsheet that is preventing the file from uploading? Did the latest version of the code change from last week?

**Reply by hanars** [reply]
ah my mistake, I though this was someone running their own instance of seqr. In the future, it is crucial that when you submit issues you need to select either the "bug report" or "feature request" issue template and fill out all the fields you are prompted for: https://github.com/broadinstitute/seqr/issues/new/choose

**Reply by hanars** [reply]
I am closing this  discussion now. Please submit a bug report issue with all fields properly filled out and then I can address the issue there
